pootle/core/utils/timezone.py

```diff
@@ -46,5 +52,8 @@ def aware_datetime(*args, **kwargs):
     """Creates a `datetime` object and makes it timezone-aware.
 
     :param args: arguments passed to `datetime` constructor.
+    :param tz: timezone in which the `datetime` should be constructed. Note that
+        this bypasses passing `tzinfo` to the `datetime` constructor, as it is
+        known not to play well with DST (unless only UTC is used).
     """
-    return make_aware(datetime.datetime(*args, **kwargs))
+    return make_aware(datetime.datetime(*args), tz=kwargs.pop('tz', None))
```

Comment:
We are loosing the ability to pass the `kwargs`. Please pop out the tz first, and keep passing the kwargs to datetime.